fv3core/stencils/fvtp2d.py

```diff
@@ -84,31 +99,32 @@ def __init__(self, namelist, hord):
             q_i_stencil,
             origin=self.grid.full_origin(add=(0, 3, 0)),
             domain=self.grid.domain_shape_full(add=(0, -3, 1)),
+            **stencil_kwargs,
```

Comment:
this has changed to be a FrozenStencil in master...

Reply:
Merged. I thought I merged before you reviewed, but maybe I hadn't pushed.